extension-manifest-v3/src/pages/panel/views/home.js

```diff
@@ -242,5 +290,22 @@ export default {
         </section>
       `}
     </template>
+  `.css`
+    .stats {
+      background-color: #F0F2F7;
+    }
+    .stats-number {
+      font-size: 24px;
```

Comment:
We are using only pre-defined fonts from our token database. Please don't make direct CSS.